releasenotes/notes/addProfileNames.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: release-notes/v2
```

Comment:
I don't think a release note entry is necessary for this...

Reply:
I was hesitant at first, but it does change visible user behavior in that these are the instructions that users are directed to from Istio.io for installing profiles.

Reply:
Looking at our release notes docs though, release notes are required for:

    CLI changes
    API changes
    Configuration schema change
    Behavioral change
    Change in non-functional attributes such as efficiency or availability, availability of a new platform
    A warning about a deprecation
    Fix of a previous Known Issue
    Promoting features

This might be considered a CLI change, but I'd lean towards agreeing that there are no release notes needed.

Reply:
The release notes should have gone in with the PR that added the vendor profile. Since it didn't, I added it here instead. I think a release note is definitely needed, even if we decide as a community to store this information on istio.io (which makes sense to me).